src/containers/ViewerPanel/index.tsx

```diff
@@ -260,7 +264,7 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         });
     }
 
-    public receiveTimeChange(timeData: TimeData) {
+    public receiveTimeChange(timeData: TimeData): any {
```

Comment:
this return should probably be void? but not any

Reply:
done